tgui/packages/tgui/interfaces/HumanInventory/index.tsx

```diff
@@ -0,0 +1,70 @@
+import { toTitleCase } from 'common/string';
+import { useBackend } from '../../backend';
+import { Button, LabeledList, Section, Stack } from '../../components';
+import { Window } from '../../layouts';
+import { HumanInventoryData, HumanInventorySlot } from './types';
+
+export const HumanInventory = (_props, context) => {
+  const { data, act } = useBackend<HumanInventoryData>(context);
+
+  return (
+    <Window width={300} height={490} title={data.name}>
+      <Window.Content>
+        <Stack fill vertical>
+          <Stack.Item grow>
+            <Section scrollable fill>
+              <LabeledList>
+                {data.slots.map((slot) => (
+                  <Slot key={slot.slot} {...slot} />
+                ))}
+              </LabeledList>
+            </Section>
+          </Stack.Item>
+          {Boolean(data.handcuffed || data.canSetInternal || data.internal) && (
+            <Stack.Item>
+              <Section>
+                {Boolean(data.handcuffed) && <Button onClick={() => act('handcuff')}>Remove handcuffs</Button>}
+                {Boolean(data.canSetInternal) && <Button onClick={() => act('internal')}>Set internals</Button>}
+                {Boolean(data.internal) && <Button onClick={() => act('internal')}>Remove internals</Button>}
+              </Section>
+            </Stack.Item>
+          )}
+        </Stack>
+      </Window.Content>
+    </Window>
+  );
+};
+
+const SlotNames = {
+  [1]: 'Back',
+  [2]: 'Mask',
+  [4]: 'Left Hand',
+  [5]: 'Right Hand',
+  [6]: 'Belt',
+  [7]: 'ID',
+  [8]: 'Ears',
+  [9]: 'Eyes',
+  [10]: 'Gloves',
+  [11]: 'Head',
+  [12]: 'Shoes',
+  [13]: 'Outer Suit',
+  [14]: 'Uniform',
+  [15]: 'Left Pocket',
+  [16]: 'Right Pocket',
+  [18]: 'Backpack',
+};
```

Comment:
```suggestion
const SlotNames = {
  [1]: '🎒 Back',
  [2]: '🤿 Mask',
  [4]: '🤚 Left Hand',
  [5]: '✋ Right Hand',
  [6]: '🩲 Belt',
  [7]: '🎫 ID',
  [8]: '🦻 Ears',
  [9]: '🥽 Eyes',
  [10]: '🧤 Gloves',
  [11]: '⛑ Head',
  [12]: '👞 Shoes',
  [13]: '🦺 Outer Suit',
  [14]: '👔 Uniform',
  [15]: '↪ Left Pocket',
  [16]: '↩ Right Pocket',
  [18]: '🎒 Backpack',
};
```

Reply:
since when is left on my right and right on my left - i'd rather we just use the letters L and R for less confusion
also backpack emoji twice??

Reply:
If you turn your palms towards you it'll make more sense, but yea L and R is fine too

Also I dunno when back and back pack display, I thought it was always just the "back" slot?